src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+public class RightTriangle extends Shape implements Figure {
```

Comment:
you may implement the interface in the Shape class, this way it will implemented in the child classes by default